bin/disk-init

```diff
@@ -23,7 +23,7 @@ if [[ ! $(ls "${loopdev}"*1) ]]; then
     echo "WARNING: Missing a required partition. This can happen once."
     echo "Please run script once more to fix."
     echo "------------------------------------------"
-    kill 0
+    exit
```

Comment:
So one thing that worries me - does this exit keep a running container around? Whereas kill 0 does not?

Reply:
Also is this missing loopback partition issue documented anywhere else?